examples/ai-billing/app/credits/stripe/success/route.ts

```diff
@@ -0,0 +1,45 @@
+import { revalidate } from "@/app/revalidate";
+import { NextRequest, NextResponse } from "next/server";
+import Stripe from "stripe";
+import { createKey, listKeys, updateKey } from "./keys";
+
+export async function GET(request: NextRequest, _response: Response) {
+  const url = new URL(request.url);
+  const session_id = url.searchParams.get("session_id") as string;
+
+  if (!session_id) {
+    return new Response("No session_id", { status: 400 });
+  }
+
+  const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!, {
+    typescript: true,
+  });
+
+  const session = await stripe.checkout.sessions.retrieve(session_id);
+
+  if (!session) {
+    return new Response("No session", { status: 400 });
+  }
+
+  const keys = await listKeys();
+  if (!keys.length) {
```

Comment:
I tripped over this, what do you think about making it clearer for potentially new users?

```suggestion
  if (keys.length === 0) {
```